tests/packagedcode/data/podspec/Starscream/Starscream.podspec.expected.json

```diff
@@ -0,0 +1,43 @@
+{
+  "type": "podspec",
+  "namespace": null,
+  "name": "Starscream",
+  "version": "4.0.3",
+  "qualifiers": {},
+  "subpath": null,
+  "primary_language": "Swift",
+  "description": null,
+  "release_date": null,
+  "parties": [
+    {
+      "type": null,
+      "role": "author",
+      "name": "Dalton Cherry, Austin Cherry",
+      "email": "http://daltoniam.com, http://austincherry.me",
+      "url": null
+    }
+  ],
+  "keywords": [],
+  "homepage_url": null,
```

Comment:
What about this https://github.com/nexB/scancode-toolkit/pull/2067/files#diff-7dc4cc146a65ad30177b788f8bcbf6e9R5 ?

Reply:
It is fixed now